themes/SuiteP/css/bootstrap.min.css

```diff
@@ -1764,13 +1764,13 @@ pre code {
     width: 58.33333333%}
 
 .col-sm-6 {
-    width: 50%}
+    }
```

Comment:
Editing the base twitter bootstrap file is bad practice, as all other col-sm-6 elements will be affected with this change.